tutorials/esp32_blogpost.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Hello World- ESP32 Firmware
+layout: post
+created_at: Wed Aug 29 2018
+permalink: tutorials/esp32_blogpost
+current: blog
+author: Arushi
+twitter: arushi1803
+categories:
+- student-posts
+---
+
+We’re currently aiming to make our contribution to the Tessel project. Tessel-Reach is a developing concept that we 
+recently got introduced to.<br>
```

Comment:
In markdown, you should do two carriage returns to demarcate paragraphs, instead of the `<br>` html syntax